src/pages/call-for-paper.js

```diff
@@ -56,5 +64,17 @@ export const query = graphql`
                 }
             }
         }
+        nextMeetup: allMeetupEvent(
```

Comment:
Est-ce qu'il n'y aurait pas moyen de faire cette requête au niveau de `src/layout` pour ne pas avoir à la déclarer dans toutes les pages ou l'on affiche le `SideMenu` ? (en injectant le `nextMeetup` dans les props du `children` dans le genre de ce qu'ils font ici https://stackoverflow.com/questions/42231074/react-passing-state-with-this-props-children)

Reply:
Tu as totalement raison, j'ai tilté après mais le réveillon est arrivé xD
Fait dans le commit 70d4047953fe4e0de2f7ef88c13659bb7b7da9cd